index.html

```diff
@@ -193,6 +197,18 @@ <h2>Implementation Details</h2>
               A fetch.txt file is allowed within the bag. Default: `true`
             </p>
           </li>
+          <li>
+            `Fetch.txt-Required`: `true`|`false`
+            <p>
+              A fetch.txt file is required. Specified when `Allow-Fetch.txt` is `true`. Default is `false`.
+            </p>
+          </li>
+          <li>
+            `Data-Empty`: `true`|`false`
+            <p>
+              Whether or not the `/data` directory should be empty. Default is `false`.
```

Comment:
I think this could be made stronger; e.g,.

    Whether or not the `/data` directory must be empty.  Default is `false`.